docs/usage.md

```diff
@@ -0,0 +1,36 @@
+This file collects some common use cases for this tool.
+
+# Generate schema-c data from integration package fields
+
+To do this, use the `generate` command. This command targets a specific dataset within an integration package at a specific version.
+
+You can pass a local Fields configuration file.
+
+`go run main.go generate <package> <dataset> <version> --tot-size <quantity>`
+
+`package`, `dataset` and `version` are mandatory. `--tot-size` is mandatory.
+
+**Example**:
+
+```shell
+$ go run main.go generate aws dynamodb 1.14.0 -t 1000 --config-file config.yml
+File generated: /path/to/corpora/1649330390-aws-dynamodb-1.14.0.ndjson
+```
+
+# Generate schema-b data from a template
+
+To do this, use the `generate-with-template` command. This command targets a specific template within the `assets/templates` folder.
```

Comment:
> This command targets a specific template within the `assets/templates` folder.

```
A body of templates, fields definition and fields definition configuration are already available in the `assets/templates` folder. You can rely on them or write your own ones.
Please consider opening a PR adding your custom templates, fields definition and fields definition configuration to the existing body, if they belong to an integration, so that we can enrich the catalogue for everyone.
```

Reply:
I'd say it does not really targets what's in the `assets/templates` folder, like the same way the `generate` command targets a specific dataset within an integration package at a specific version :)

See my suggestion above